muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/evaluation/EvaluationRESTService.java

```diff
@@ -1765,5 +1771,13 @@ private RestInterimEvaluationRequest toRestModel(InterimEvaluationRequest interi
         interimEvaluationRequest.getRequestText(),
         interimEvaluationRequest.getArchived());
   }
+  
+  private Boolean getHasPedagogyForm(String studentIdentifier) {
```

Comment:
This method seems to have been copy/pasted to several classes. Couldn't we simply have...

```
public boolean hasPedagogyForm(String studentIdentifier) {
  return findFormByStudentIdentifier(studentIdentifier) != null;
}

```
...in `PedagogyController` instead?